packages/core/EventManager.ts

```diff
@@ -338,9 +338,18 @@ export default class EventManager {
   private async createAllCalendarEvents(event: CalendarEvent) {
     let createdEvents: EventResult<NewCalendarEventType>[] = [];
     if (event.destinationCalendar && event.destinationCalendar.length > 0) {
-      for (const destination of event.destinationCalendar) {
-        if (destination.credentialId) {
-          let credential = this.calendarCredentials.find((c) => c.id === destination.credentialId);
+      // Want to only create one event per calendar app as they sync across their own platforms
+      const destinationCalendarApps = event.destinationCalendar.reduce((apps, calendar) => {
+        if (!apps.includes(calendar.integration)) apps.push(calendar.integration);
+        return apps;
+      }, [] as string[]);
```

Comment:
We only want to create one event per calendar app platform. This is because if we add the correct calendar ids then those events will be pushed to other calendars on the same platform. 

I tested this on a team that contained GCal and Outlook integrations and events created on GCal did not get pushed to Outlook.

Reply:
I think we should leave this as it is, we better support every host calendar app. And we just do better filtering with Google that has enhanced behaviour for existing users when being added on an event.

Reply:
Tested this with two Outlook calendars and it didn't duplicate events. There was still a duplicate event being created with two GCals so I'll change the logic to only create one event for only GCal.